src/main/java/frc/robot/RobotContainer.java

```diff
@@ -191,6 +195,14 @@ private void configureButtonBindings() {
         drivetrain, shooterSubsystem, turretSubsystem, ledSubsystem, controlBindings.translationX(),
         controlBindings.translationY(), STOCKPILE_MID_RED, STOCKPILE_MID_BLUE, STOCKPILE_INTERPOLATOR, 0.3)));
     
+    controlBindings.babyBomber().ifPresent(trigger -> trigger.whileTrue(
+      new BabyBirdCommand(
+        turretSubsystem, shooterSubsystem).deadlineWith(new LEDBlinkCommand(ledSubsystem, FLASH_COLOR, 0.1))
+      .andThen(new ShootTeleopCommand(
+        drivetrain, shooterSubsystem, turretSubsystem, ledSubsystem, controlBindings.translationX(),
+        controlBindings.translationY(), STOCKPILE_MID_RED, STOCKPILE_MID_BLUE, STOCKPILE_INTERPOLATOR, 0.3)
+        .deadlineWith(new LEDAlternateCommand(ledSubsystem, Color.kBlue, Color.kOrange, Seconds.of(0.1))))));
```

Comment:
This is OK, but my preference would be to move the construction of `BabyBirdCommand` and `ShootTeleopCommand` into the `AutoCommands` class and then reuse them for both bindings instead of doing copy-paste.

Reply:
I just pushed a commit that moves them, however I'm unsure on how to resolve the error as the autocommands object is created before controlbindings is initialized in the constructor.

Reply:
I think a good way to handle this would be to have the control bindings passed to the `shootMid()` command instead of the constructor. This should work well since a new command instance is created each time the method is called.